src/api/search/index.ts

```diff
@@ -21,7 +21,7 @@ export function convertToSearchNodePayload(
   criteria: SearchCriteria,
   organizationId?: number
 ): SearchNodePayload | undefined {
-  if (Object.keys(criteria).length === 0) {
+  if (Object.keys(criteria).length === 0 && !organizationId) {
```

Comment:
this code was previously
```
if (criteria === {}) {
  return undefined;
}
```
That check was incorrect as well for not considering the org id...  we got lucky since it never tested true.